idaplugin/rematch/actions/base.py

```diff
@@ -13,9 +13,16 @@ def plugin_path(*path):
 class Action(object, idaapi.action_handler_t):
   """Actions are objects registered to IDA's interface and added to the
   rematch menu and toolbar"""
+  dialog = None
+
+  reject_handler = None
```

Comment:
I might be bitching here, but won't it be better to implement a "Not yet ready" function instead of setting to to None ?